pypowerwall/local/pypowerwall_local.py

```diff
@@ -165,13 +165,13 @@ def poll(self, api: str, force: bool = False,
                 self.pwcachetime[api] = time.perf_counter() + 600
                 self.pwcache[api] = None
                 return None
-            if r.status_code == 429:
+            elif r.status_code == 429:
                 # Rate limited - Switch to cooldown mode for 5 minutes
                 self.pwcooldown = time.perf_counter() + 300
                 log.error('429 Rate limited by Powerwall API at %s - Activating 5 minute cooldown' % url)
-                # Serve up cached data if it exists
+                # Serve up cached data if it exists (@emptywee: this doesn't look like we serve up cached data here?)
```

Comment:
This comment needs to be removed.  If we hit this, we are at the TTL for the cached objects which is what the user expects us to do. After that TTL we should send fresh data or return None indicating something bad happened or no connection.